CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ x.x.x
 **ENHANCEMENTS**
 - Add support for multiple Elastic File Systems.
 - Add support for multiple FSx File System.
+- Set AuthInfo=cred_expire=70 to allow job faster requeue when nodes are not available.
```

Comment:
- Slurm: Set `AuthInfo=cred_expire=70` to reduce the time requeued jobs must wait before starting again when nodes are not available.